Dockerfile

```diff
@@ -1,5 +1,5 @@
 # This is a multi-stage build which requires Docker 17.05 or higher
-FROM python:3.7-alpine as molecule-builder
+FROM python:alpine3.8 as molecule-builder
```

Comment:
```suggestion
FROM python:3.7-alpine3.8 as molecule-builder
```

Reply:
This would make it more clear which version of Python and alpine is used (since the version numbers are quite close).

Also: Why alpine 3.8 and not 3.9?

Reply:
Alpine 3.9 has more and higher vulnerabilities.

Reply:
I don't know too how to edit pull request and I am lazy to solve it out. :)

Reply:
Could you be a bit more specific and show some sources for this statement please?

Reply:
https://hub.docker.com/_/python/ doesn't seem to show any data about vulnerabilities to me. Which ones are the ones relevant to molecule?

About editing pull requests: You'll need to amend your commit and do a force push of your branch (you'll need to do that anyways, since `--signoff` is required to accept the DCO).

Reply:
@cleveritcz just hit "Commit suggestions" on GitHub UI.

Reply:
Good luck with the signoff requirement...

Reply:
You can manually type it anyway